apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -0,0 +1,823 @@
+/**
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Update license header 
```suggestion
 * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```